worldengine/drawing_functions.py

```diff
@@ -411,57 +369,58 @@ def draw_ancientmap(world, target, resize_factor=1,
 
     land_color = (
         181, 166, 127, 255)  # TODO: Put this in the argument list too??
-    borders = _find_land_borders(world, resize_factor)
-
-    if draw_outer_land_border:
-        outer_borders = _find_outer_borders(world, resize_factor, borders)
-        outer_borders = _find_outer_borders(world, resize_factor, outer_borders)
 
-    if draw_mountains:  # TODO: numpy offers masked arrays - maybe they can be leveraged for all this?
-        mountains_mask = _find_mountains_mask(world, resize_factor)
 
-    if draw_biome:
-        biome_masks = _build_biome_group_masks(world, resize_factor)
 
-        # TODO: there was a stub for a rock desert biome group here
-        # it should be super easy to introduce that group with the new
-        # biome group concept but since it did nothing I removed the stub
-
-    def unset_mask(pos):
-        x, y = pos
-        mountains_mask[y, x] = 0
-
-    def unset_boreal_forest_mask(pos):
-        x, y = pos
-        biome_masks['boreal forest'][y, x] = 0
+    scaled_ocean = world.ocean.repeat(resize_factor,0).repeat(resize_factor,1)
```

Comment:
super nitpick (shame on me), maybe you could put a space after the commas and between the `>` sign below

Reply:
OK, I'm giving up and installing pylint :)

Reply:
PyCharm IDE is also free... unless you're an vim or emacs fan, then yeah, pylint and pep8 are your friends. :)

Reply:
I'm using Visual Studio and don't want to change my workflow, if I can help it. So kindof the opposite of vim :)
It has built-in formatting I think, but that has to be triggered manually, which I maybe just should remember to do. It also has builtin pylint support which I wanted to give a try ... mixed results so far.